src/backend/joanie/core/api/client/__init__.py

```diff
@@ -581,6 +581,45 @@ def submit_installment_payment(self, request, pk=None):  # pylint: disable=unuse
 
         return Response(payment_infos, status=HTTPStatus.OK)
```

Comment:
In commit body (07841269230ca0b38472b5cb95797e8125c2dd9e):

`s/neeeded/needed`